GoogleMapsApi/Entities/Places/Request/PlacesRequest.cs

```diff
@@ -54,12 +53,11 @@ protected internal override string BaseUrl
 		public RankBy RankBy { get; set; }
 
 		/// <summary>
-		/// Restricts the results to Places matching at least one of the specified types. 
-		/// Types should be separated with a pipe symbol (type1|type2|etc). 
+		/// Restricts the results to Places matching the specified type. 
 		/// See the list of supported types - https://developers.google.com/maps/documentation/places/supported_types
 		/// 
 		/// </summary>
-		public string Types { get; set; }
+		public string Type { get; set; }
```

Comment:
Is Google's API change is backward compatible? If yes you can keep the old property name and use ```[DataMember(Name = "types")]```
This way you won't break users if the library